docs/subvolume.md

```diff
@@ -0,0 +1,46 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete` : [delete](#delete) stale subvolumes after user's confirmation.
```

Comment:
How about if on the first line we show the args, then only below describe them?
```suggestion
* `delete <subvolumes> <filesystem> <subvolumegroup>` :  
     [delete](#delete) stale subvolumes after user's confirmation.
```

Reply:
And do we really need to support multiple subvolumes at the same time? Seems like one at a time is sufficient for a CLI command. They can easily wrap it with a bash loop if they want. Perhaps more helpful is when we implement the option to delete all stale volumes, then no need even to provide their name on the CLI.